source/_patterns/02-components/accordion/accordion.scss

```diff
@@ -10,7 +10,7 @@
 	summary {
 		@include icon(glyph(chevron-right), 24, "outline", $partial: $partial);
 
-		height: to-rem($pxValue: 48);
+		padding-block: $db-spacing-sm;
```

Comment:
This is only available in base-v3 right? So this branch should point on v3 as well?

Reply:
It's even already part of the current release, spacings have bern defined even already for Guidelines 2.0: https://github.com/db-ui/base/blob/main/tokens/spacings.json#L11L14